pyrq/unique_queues.py

```diff
@@ -18,18 +19,19 @@ def _create_chunks(items):
         yield chunk
 
 
-class Queue(object):
+class UniqueQueue(object):
     """
-    Queue is a simple queue implemented using List as the queue, multiple Lists as the processing queues and a Hash as
+    UniqueQueue is a queue implemented using List as the queue, multiple Lists as the processing queues and a Hash as
     a storage for processing queue timeouts (so you can tell which processing queue is expired).
+    UniqueQueue keeps the uniqueness data in the queue therefore if you add the same item more times the one adds it to
```

Comment:
_"UniqueQueue items are always unique, adding the same item more than once will be ignored"_